components/lib/confirmdialog/ConfirmDialog.js

```diff
@@ -81,6 +82,7 @@ export const ConfirmDialog = React.memo(React.forwardRef((props, ref) => {
         return () => {
             OverlayService.off('confirm-dialog', confirm);
         }
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
@mertsincan to make these pass the lint rules I had to do this but wasn't sure if you wanted to fix them all the "right" way to not have any of these eslint ignores.

Reply:
@melloware It can stay that way for now. We can improve on this in future versions. Thanks a lot!